src/app/newcomers/nonCodeMethodsData.ts

```diff
@@ -0,0 +1,43 @@
+import { MethodsFormat } from './nonCodeMethods';
+
+export const nonCodeMethodsData: MethodsFormat[] = [
+    {
+        methodName: 'Documentation',
+        image: '/assets/img/documentation.jpg',
+        // tslint:disable-next-line:max-line-length
+        details: 'Documentation is a huge part of a successful open source project. Organizations need documentation to help attract users and new contributors.'
+    },
+    {
+        methodName: 'Research',
+        image: '/assets/img/research.jpg',
+        details: 'An organization may need help crunching numbers or analyzing feedback to better understand what the users want or need.'
+    },
+    {
+        methodName: 'Outreach',
+        image: '/assets/img/outreach.jpg',
+        // tslint:disable-next-line:max-line-length
+        details: 'Outreach is how open source projects attract new users and developers. This includes writing blog posts or making videos, organizing meetups, or helping others learn about the project.'
+    },
+    {
+        methodName: 'Training',
+        image: '/assets/img/training.jpg',
+        details: 'Teach others how to use the project. This is a specialized form of outreach or documentation.'
+    },
+    {
+        methodName: 'User Interface',
+        image: '/assets/img/UI.jpg',
+        // tslint:disable-next-line:max-line-length
+        details: 'User interface can include many types of tasks including designing new visual aspects of a webpage or creating a new logo for the project. It may also include various accessibility opportunities to help make the project easier for people who are visually impaired.'
+    },
+    {
+        methodName: 'Quality Assurance',
+        image: '/assets/img/qualityAssurance.jpg',
+        details: 'Finding and verifying bugs is an important part of software development.'
+    },
+    {
+        methodName: 'Other',
+        image: '/assets/img/others.jpg',
+        // tslint:disable-next-line:max-line-length
+        details: 'If you have an idea that may help an organization, reach out to them and let them know! Sometimes the best task is one that the organization hasn’t even considered yet.iil/'
```

Comment:
`iil/` possible typo?

Reply:
Yes. Thanks for pointing it out. @sara-02 :)